src/MOI/indicator_constraint.jl

```diff
@@ -0,0 +1,117 @@
+
+function MOI.supports_constraint(
+    ::Optimizer, ::Type{MOI.VectorAffineFunction{Float64}}, ::Type{<:MOI.IndicatorSet{A, S}}
+) where {A, S <: Union{MOI.LessThan{Float64}, MOI.GreaterThan{Float64}, MOI.EqualTo{Float64}}}
```

Comment:
Is `EqualTo` tested? 

Is there are a restriction on `A`?

Reply:
See the docstring for `MOI.IndicatorSet{A, S}`
A is a value from the enum `MOI.ActivationCondition`.:
```
julia> typeof(MOI.ACTIVATE_ON_ONE)
Enum MathOptInterface.ActivationCondition:
ACTIVATE_ON_ZERO = 0
ACTIVATE_ON_ONE = 1
```

Unfortunately, one cannot type-check value parameters of a type

Reply:
> Is EqualTo tested?

No that's a good point, it is missing from MOI, I'll add a PR with an equality test